Modules/Challenges/test/Challenges.API.Tests.Integration/Features/CreateChallenge.feature

```diff
@@ -0,0 +1,37 @@
+﻿@Integration
+Feature: Create a Challenge
+
+User creates a Challenge
+
+Scenario: Creating a Challenge as an anonymous user
+	Given the user is unauthenticated
+	When a POST request is sent to the Challenges endpoint
+	Then the response status code is 201
+	And the response contains a Challenge
+	And the Challenge does not contain information about the creator
+
+Scenario: Creating a Challenge as an authenticated user
+	Given the user is authenticated
+	When a POST request is sent to the Challenges endpoint
+	Then the response status code is 201
+	And the response contains a Challenge
+	And the Challenge contains information about the creator
+
+Scenario: Creating a Challenge with a JSON sent in the request content
+	When a POST request is sent to the Challenges endpoint with
+		| Key  | Value                              |
+		| Body | {"this": "is some arbitrary json"} |
+	Then the response status code is 415
+
+Scenario: Creating a Challenge with an invalid JSON sent in the request content
+	When a POST request is sent to the Challenges endpoint with
+		| Key  | Value                             |
+		| Body | { "thisJSON": "has an extra }" }} |
```

Comment:
We once said that we want to use the word "content" instead of "body". In the title of this scenario you already use it, but not in the table. Please search in the whole project for the word "Body" and replace it accordingly.